flow-defs/feedback-screen.js.flow

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
Dark mode color is quite weird, is it expected @HectorSancho ?
![image](https://user-images.githubusercontent.com/6722153/135070596-608465cd-ed90-4ba1-83e4-9e648fec1a47.png)

Headers works different in this case

---

A 1px white appear around de componente
![image](https://user-images.githubusercontent.com/6722153/135070796-2877c520-7bb0-412a-afa8-ed2248923a00.png)

--

Would be nice to have the distribution of buttons horizontally in mobile, instead of vertically. (I suppose that this depends on button layout)

Reply:
The background color in DarkMode for desktop is the one that is set by Boxed component. Is that the one that seems weird to you?

Reply:
the 1px transparent was intentional because of this visible pixel around the image that does not appear in the specs:
https://github.com/Telefonica/mistica-web/pull/343/commits/84fe0f67cc75ecbd71b5a224cdc3fb58f76dd7e6#diff-da49445e16cc5dcbb9554ebb7566091f80c2bf044af0fc5f7841a6e19278e816

We have implemented that Boxed component in inverse mode has a `border: 1px solid transparent` and `backgroundClip: padding-box`.

Do you think it is not correct?

Reply:
the distribution of buttons in mobile is given by ButtonFixedFooterLayout. This occurrs in this component since it is fullscreen. I have another PR for the "header" case Hector designed: #345. In that case, the buttons are aligned if they have enough space.

Reply:
1. We can use boxed with isInverse, no problem.
2. We talked about transparent pixel and we are going to remove it

Reply:
done!